src/reanimated2/core.js

```diff
@@ -150,3 +151,33 @@ export function startMapper(mapper, inputs = [], outputs = []) {
 export function stopMapper(mapperId) {
   NativeReanimated.stopMapper(mapperId);
 }
+
+export let runOnJS = (fun) => fun;
+
+if (Platform.OS !== 'web') {
+  runOnJS = (fun) => {
+    'worklet';
+    if (!_WORKLET) {
+      // eslint-disable-line
+      return fun;
+    }
+    if (!fun.__callAsync) {
+      throw new Error(
+        "Attempting to call runOnJS with an object that is not a host function. Using runOnJS is only possible with methods that are defined on the main React-Native Javascript thread and that aren't marked as worklets"
+      );
+    } else {
+      return fun.__callAsync;
+    }
+  };
+}
+
+const capturableConsole = console;
+runOnUI(() => {
+  'worklet';
+  const console = {
+    log: runOnJS(capturableConsole.log),
+    warn: runOnJS(capturableConsole.warn),
+    error: runOnJS(capturableConsole.error),
+  };
+  _globalSetter('console', console); // eslint-disable-line
```

Comment:
Can we do `global.console = console` here? That's how you'd do it in regular JS so I'd prefer to use `global` here to keep it consistent. If not maybe we can enable the use of `global` instead of adding new function?

Reply:
That was my first approach but unfortunately, it's not working.